nav2_regulated_pure_pursuit_controller/src/regulated_pure_pursuit_controller.cpp

```diff
@@ -447,13 +464,27 @@ bool RegulatedPurePursuitController::inCollision(const double & x, const double
     return false;
   }
 
-  unsigned char cost = costmap_->getCost(mx, my);
+  // if the robot is even potentially in an inscribed collision
+  footprint_cost_ = _collision_checker->pointCost(wx, wy);
```

Comment:
Remove all the logic around pointCost for the same rational I gave on `possible_inscribed_cost_`, we just need to check the footprint only here!

Reply:
what about this one <https://github.com/ros-planning/navigation2/blob/2e6594ca9d7f6652c914f5b5961324bfc15b9e82/nav2_regulated_pure_pursuit_controller/src/regulated_pure_pursuit_controller.cpp#L473>

Reply:
Nope, thanks for asking, keep that one. that's for the heuristics, that's OK as is due to inflation